src/Person.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Person = ({
+  name,
+  age,
+  partner,
+}) => (
+  <>
+    <h2>{`${name}`}</h2>
+    <p>{`${age}`}</p>
+    <p>{`${partner}`}</p>
```

Comment:
You should render info about person in the next format
<section className="Person">
  <h2 className="Person__name">My name is Alex</h2>
  <p className="Person__age">I am 25</p>
  <p className="Person__partner">I am not married</p>
</section>